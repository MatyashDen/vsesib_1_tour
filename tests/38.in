25 11 12 9
1 3
4 4
6 10
7 3
13 7
15 5
22 5
24 7
24 9
