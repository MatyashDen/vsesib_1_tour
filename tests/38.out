472
0RLE0LRE0DUE0UDE0RRLLE0RDULE0RUDLE0LLRRE0LDURE0LUDRE0DDUUE0RRRLLLE0RRDULLE0RRUDLLE0LLLRRRE0LLDURRE0LLUDRRE0LDDUURE0RRRRLLLLE0RRRDULLLE0RRRUDLLLE0LLLDURRRE0LLLUDRRRE0LLDDUURRE0RRRRRLLLLLE0RRRRDULLLLE0RRRRUDLLLLE0LLLDDUURRRE0RRRRRRLLLLLLE0RRRRRDULLLLLE0RRRRRUDLLLLLE0RRRRRRUDLLLLLLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
1RLE1LRE1DUE1UDE1RRLLE1RDULE1RUDLE1LLRRE1LDURE1DDUUE1RRRLLLE1RRDULLE1RRUDLLE1RDDUULE1LLLRRRE1LLDURRE1RRRRLLLLE1RRRDULLLE1RRRUDLLLE1RRDDUULLE1LLLLRRRRE1LLLDURRRE1RRRRUDLLLLE1LLLLDURRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
2LRE2DUE2UDE2LLRRE2LDURE2LUDRE2DDUUE2UUDDE2LLLRRRE2LLDURRE2LLUDRRE2LDDUURE2LUUDDRE2DDDUUUE2UUUDDDE2LLLDURRRE2LLDDUURRE2LDDDUUURE2LUUUDDDRE2DDDDUUUUE2UUUUDDDDE2LLLDDUURRRE2LLDDDUUURRE2LDDDDUUUURE2LUUUUDDDDRE2DDDDDUUUUUE2UUUUUDDDDDE2UUUUUUDDDDDDEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
3RLE3LRE3DUE3UDE3RRLLE3RDULE3LLRRE3LDURE3LUDRE3DDUUE3RRRLLLE3RRDULLE3RDDUULE3LLLRRRE3LLDURRE3LLUDRRE3LDDUURE3DDDUUUE3RRRDULLLE3RRDDUULLE3RDDDUUULE3LLLDURRRE3LLLUDRRRE3LLDDUURRE3LDDDUUURE3DDDDUUUUE3RRRDDUULLLE3RRDDDUUULLE3RDDDDUUUULE3LLLDDUURRRE3LLDDDUUURRE3LDDDDUUUURE3DDDDDUUUUUE3LLLDDDUUURRRE3LLDDDDUUUURRE3LDDDDDUUUUURE3LLLDDDDUUUURRRE3LLDDDDDUUUUURRE3LLLDDDDDUUUUURRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
4RLE4LRE4DUE4UDE4RRLLE4RDULE4RUDLE4LLRRE4LDURE4LUDRE4DDUUE4UUDDE4RRRLLLE4RRDULLE4RRUDLLE4RDDUULE4RUUDDLE4LLLRRRE4LLUDRRE4LUUDDRE4DDDUUUE4UUUDDDE4RRRDULLLE4RRRUDLLLE4RRDDUULLE4RRUUDDLLE4RDDDUUULE4RUUUDDDLE4LLLLRRRRE4LLLUDRRRE4LLUUDDRRE4LUUUDDDRE4DDDDUUUUE4UUUUDDDDE4RRRDDUULLLE4RRDDDUUULLE4RDDDDUUUULE4LLLLLRRRRRE4DDDDDUUUUUE4RRRDDDUUULLLE4RRDDDDUUUULLE4RDDDDDUUUUULE4LLLLLLRRRRRRE4RRRDDDDUUUULLLE4LLLLLLLRRRRRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
5RLE5LRE5DUE5UDE5RDULE5LLRRE5LDURE5LUDRE5DDUUE5RDDUULE5LLLRRRE5LLDURRE5LLUDRRE5LDDUURE5DDDUUUE5RDDDUUULE5LLLLRRRRE5LLLDURRRE5LLLUDRRRE5LLDDUURRE5LDDDUUURE5LLLLLRRRRRE5LLLLDURRRRE5LLLLUDRRRRE5LLLDDUURRRE5LLDDDUUURRE5LLLLLDURRRRRE5LLLLLUDRRRRRE5LLLLDDUURRRRE5LLLDDDUUURRRE5LLLLLDDUURRRRRE5LLLLDDDUUURRRRE5LLLLLDDDUUURRRRREWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
6RLE6LRE6DUE6UDE6RRLLE6RDULE6RUDLE6LLRRE6LDURE6LUDRE6DDUUE6UUDDE6RRRLLLE6RRUDLLE6RUUDDLE6LLLRRRE6LLDURRE6LLUDRRE6LDDUURE6LUUDDRE6UUUDDDE6RRRUDLLLE6RRUUDDLLE6RUUUDDDLE6LLLLRRRRE6LLLDURRRE6LLLUDRRRE6LLDDUURRE6LLUUDDRRE6LUUUDDDRE6RRRUUDDLLLE6RRUUUDDDLLE6LLLLLRRRRRE6LLLLDURRRRE6LLLLUDRRRRE6LLLDDUURRRE6LLLUUDDRRRE6LLUUUDDDRRE6RRRUUUDDDLLLE6LLLLLDURRRRRE6LLLLLUDRRRRRE6LLLLDDUURRRRE6LLLLUUDDRRRRE6LLLUUUDDDRRRE6LLLLLDDUURRRRRE6LLLLLUUDDRRRRRE6LLLLUUUDDDRRRRE6LLLLLUUUDDDRRRRRE
7RLE7LRE7UDE7RUDLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
8RLE8UDE8RUDLE8UUDDE8RUUDDLE8UUUDDDE8RUUUDDDLE8UUUUDDDDE8RUUUUDDDDLE8UUUUUDDDDDE8RUUUUUDDDDDLE8UUUUUUDDDDDDE8RUUUUUUDDDDDDLEWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
